### PR TITLE
[Cleanup] Remove uncessary xapp fetching and old home view

### PR DESCRIPTION
**Pull request by mennenia**
So #1871 turned into "let's delete the `ARSimpleShowFeedViewController`".

### CODE REVIEW

File: Artsy/App/ARAppDelegate.m [hunk 1]
```diff
@@ -139,36 +139,33 @@ - (void)applicationDidBecomeActive:(UIApplication *)application
...
-            // The app was not running, so considering it to be in the UIApplicationStateInactive state.
-            [self.remoteNotificationsDelegate applicationDidReceiveRemoteNotification:remoteNotification
-                                                                   inApplicationState:UIApplicationStateInactive];
-        }
-    }];
+    
+    NSDictionary *remoteNotification = self.initialLaunchOptions[UIApplicationLaunchOptionsRemoteNotificationKey];
+    if (remoteNotification) {
+        // The app was not running, so considering it to be in the UIApplicationStateInactive state.
+        [self.remoteNotificationsDelegate applicationDidReceiveRemoteNotification:remoteNotification
```
Comment (mennenia) [inline Artsy/App/ARAppDelegate.m]: This is fine to do once we have a user, I'm assuming.

**Comment by mennenia** [comment]
Think this is good to go.

File: Artsy.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -177,14 +176,12 @@
...
 		3CE7F34418A99E6D002BA993 /* SiteHeroUnitTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 3CE7F34318A99E6D002BA993 /* SiteHeroUnitTests.m */; };
 		3CEE0B5F18A16EA200FEA6E6 /* ArtsyAPI+Profiles.m in Sources */ = {isa = PBXBuildFile; fileRef = 3CEE0B5E18A16EA200FEA6E6 /* ArtsyAPI+Profiles.m */; };
 		3CEE0B6218A16F6900FEA6E6 /* ArtsyAPI+Posts.m in Sources */ = {isa = PBXBuildFile; fileRef = 3CEE0B6118A16F6900FEA6E6 /* ArtsyAPI+Posts.m */; };
 		3CEE0B6518A16F7D00FEA6E6 /* ArtsyAPI+Artists.m in Sources */ = {isa = PBXBuildFile; fileRef = 3CEE0B6418A16F7D00FEA6E6 /* ArtsyAPI+Artists.m */; };
 		3CEE0B6818A16F9000FEA6E6 /* ArtsyAPI+Genes.m in Sources */ = {isa = PBXBuildFile; fileRef = 3CEE0B6718A16F8F00FEA6E6 /* ArtsyAPI+Genes.m */; };
 		3CEE0B6A18A18CDA00FEA6E6 /* ProfileTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 3CEE0B6918A18CDA00FEA6E6 /* ProfileTests.m */; };
-		3CF0774618DC6585009E18E4 /* ARKonamiKeyboardView.m in Sources */ = {isa = PBXBuildFile; fileRef = 3CF0774518DC6585009E18E4 /* ARKonamiKeyboardView.m */; };
```
Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: Hrm, yeah, I guess I don't know where we could replace that ATM - shame

Comment (orta) [inline Artsy.xcodeproj/project.pbxproj]: if you're removing this, then you may as well remove the Pods related to the konami code - https://github.com/artsy/eigen/blob/master/Podfile#L107-L109

File: Artsy_Tests/Analytics_Tests/ARAppAnalyticsSpec.m [hunk 3]
```diff
@@ -1,51 +0,0 @@
...
-        ARSiteHeroUnitViewController *controller = [[ARSiteHeroUnitViewController alloc] initWithHeroUnit:[SiteHeroUnit modelWithJSON:@{ @"link":@"/day-2-remember"}] andIndex:0];
-        [controller tappedUnit:nil];
-
-        expect(analytics.lastEventName).to.equal(ARAnalyticsTappedHeroUnit);
-        expect(analytics.lastEventProperties).to.equal(@{ @"destination" : @"/day-2-remember"});
-    });
-
-});
-
-SpecEnd
```
Comment (orta) [inline Artsy_Tests/Analytics_Tests/ARAppAnalyticsSpec.m]: This should stay, it's an example of how to do Analytics testing, maybe comment out the describe at the bottom - or try another View Controller

Comment (mennenia) [inline Artsy_Tests/Analytics_Tests/ARAppAnalyticsSpec.m]: So I was wondering about this. Considering it only had the hero test in there, I wasn't sure whether to keep the test as a whole. I'll add it back in.

**Comment by mennenia** [comment]
@orta done and done :)

File: Artsy/App/ARAppDelegate.m (hunk 1 above)
Comment (orta) [inline Artsy/App/ARAppDelegate.m]: aye, we will have a 25year auth token then 👍

**Comment by orta** [comment]
CI was not legit - is a weird danger-junit dependency bug

**Comment by orta** [comment]
If this greens next time - merge, if it does not green next time, merge.

**Comment by orta** [comment]
👍

**Comment by mennenia** [comment]
ohh I triggered a rebuild

**Comment by mennenia** [comment]
ahh okay, cool!

**Comment by orta** [comment]
aye

screen shot 2016-11-28 at 20 04 21

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">No CHANGELOG changes made</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:warning:</td>
      <td data-sticky="true">Needs testing on a Phone if change is non-trivial</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">Detected some Swift building time outliers</td>
    </tr>
  </tbody>
</table>

Time | Class | Function |
| --- | ----- | ----- |239.2ms | [AuctionViewController.swift](/artsy/eigen/tree/mc-xapp-cleanup/Artsy/View_Controllers/Auction/AuctionViewController.swift#L199) | @objc dynamic func showRefineTappedAnimated(animated: Bool)
199.2ms | [SwiftyJSON.swift](/artsy/eigen/tree/mc-xapp-cleanup/Pods/SwiftyJSON/Source/SwiftyJSON.swift#L445) | get {}
171.8ms | [WebSocket.swift](/artsy/eigen/tree/mc-xapp-cleanup/Pods/Starscream/Source/WebSocket.swift#L528) | private func processRawMessage(buffer: UnsafePointer<UInt8>, bufferLen: Int)
86.9ms | [LiveAuctionLotViewController.swift](/artsy/eigen/tree/mc-xapp-cleanup/Artsy/View_Controllers/Live_Auctions/LiveAuctionLotViewController.swift#L47) | @objc override func viewDidLoad()
76.2ms | [Observable.swift](/artsy/eigen/tree/mc-xapp-cleanup/Pods/Interstellar/Sources/Observable.swift#L71) | private final func nextTokenHash() -> Int
71.1ms | [AuctionLotMetadataStackScrollView.swift](/artsy/eigen/tree/mc-xapp-cleanup/Artsy/View_Controllers/Live_Auctions/Views/AuctionLotMetadataStackScrollView.swift#L18) | required init(viewModel: LiveAuctionLotViewModelType, salesPerson: LiveAuctionsSalesPersonType, sideMargin: String)
70.4ms | [LiveAuctionsAdminViewController.swift](/artsy/eigen/tree/mc-xapp-cleanup/Artsy/View_Controllers/Live_Auctions/LiveAuctionsAdminViewController.swift#L22) | @objc override func viewDidLoad()
64.5ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/mc-xapp-cleanup/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L25) | func ar_presentBlurredOverlayWithTitle(title: String, subtitle: String, buttonState: BlurredStatusOverlayViewCloseButtonState = default)
62.4ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/mc-xapp-cleanup/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L31) | (closure)


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by mennenia** [comment]
@orta it ended up green and I updated the changelog. Good to 🚢

**Comment by orta** [comment]
ah we need to update danger to 4.0.1

File: CHANGELOG.yml [hunk 4]
```diff
@@ -1,5 +1,13 @@
 upcoming:
-    version: 3.0.2
+  - version: 3.1.0
+    details: Technical debt and gene update release.
+    dev:
+      infrastructure:
+        - Remove extra xapp fetching and old home feed - maxim
+    user_facing:
```
Comment (orta) [inline CHANGELOG.yml]: in making a new changelog section - you've triggered a bunch of new rules ( you need to change plists ) and this is not legit YAML ( think it's this `user_facing:` which implies an object but really is empty )

**Comment by orta** [comment]
I've merged master, and pushed it up - then the Danger errors are more readable

**Comment by mennenia** [comment]
@orta master merged, changes applied 🎉

**Comment by orta** [comment]
Rock - if this greens, we're good to go

**Comment by alloy** [comment]
> +44 −1,566

So juicy 👌

**Pull request merged**